django/conf/__init__.py

```diff
@@ -109,6 +110,28 @@ def configure(self, default_settings=global_settings, **options):
             setattr(holder, name, value)
         self._wrapped = holder
 
+    @staticmethod
+    def _add_script_prefix(value):
+        """
+        Prefix header value SCRIPT_NAME to ``value`` under certain circumstances.
+
+        Useful for when the app is being served at a subpath and manually prefixing
+        that subpath to STATIC_URL and STATIC_ROOT in settings is inconvenient
```

Comment:
`STATIC_ROOT` should probably be `MEDIA_URL`? `*_ROOT` refers to actual fs paths.

Reply:
done